golang/lib/api_container_context/api_container_context.go

```diff
@@ -0,0 +1,36 @@
+package api_container_context
+
+type APIContainerContext struct {
+	containerID  string
+	ipInsideEnclave string
+	portInsideEnclave uint32
+	ipOnHostMachine string
+	portOnHostMachine uint32
+}
+
+func NewAPIContainerContext(containerID string, ipInsideEnclave string, portInsideEnclave uint32, ipOnHostMachine string, portOnHostMachine uint32) *APIContainerContext {
+	return &APIContainerContext{containerID: containerID, ipInsideEnclave: ipInsideEnclave, portInsideEnclave: portInsideEnclave, ipOnHostMachine: ipOnHostMachine, portOnHostMachine: portOnHostMachine}
+}
+
+func (A APIContainerContext) GetContainerID() string {
```

Comment:
Can we replace this `A` with something more descriptive, e.g. `apiContext`?

Reply:
Sure! I didn't put attention to the name because it was auto-generated by GoLand. Now I'll change it